lib/Doctrine/DBAL/Types/JsonType.php

```diff
@@ -43,6 +43,10 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
         if (null === $value) {
+            if ($platform->getName() == 'oracle') {
+                return '';
+            }
+
```

Comment:
If you have another idea for this, I'm all ears. It's the smallest change I can do to avoid binding `null` to a CLOB.